src/logging/logger.ts

```diff
@@ -1,29 +1,24 @@
-import { App } from 'electron';
-import logger from 'electron-log';
+import { App } from '@/app-globals';
+import logger, { LogFunctions, LogLevel } from 'electron-log'; // eslint-disable-line no-restricted-imports
```

Comment:
If you will try to `import logger from 'electron-log'`, you will get an *ESLint* error, because we want you to use our custom-defined logger.
But here we have to import the logger from `electron-log` to configure it.